server/src/dynamodb.ts

```diff
@@ -21,7 +21,7 @@ export class Dynamo implements Repository {
   }
 
   public async init() {
-    if (process.env.CREATE_DYNAMO_TABLES) {
+    if (process.env.LISTO_DATABASE === 'Dynamo' && process.env.CREATE_DYNAMO_TABLES) {
```

Comment:
Do you need this check? it'll only get run if  `init` is called, which won't happen unless this is the desired database?

Reply:
This still needs to be here for when Listo is deployed to prod in AWS, as the table is setup via a separate config (CloudFormation) right? And we dont want the code setting it up for us. This is only for local dev using the dynamo local db.

Reply:
ahh right. Hopefully we can cull that down into a single option somehow